portfolio/src/main/webapp/index.html

```diff
@@ -8,9 +8,66 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+      <h1>Pranjali Sharma</h1>
+      <div id="about">
+      <h3>About</h3>
+      <p> I am Pranjali, a rising junior at Columbia University (SEAS '22) studying Computer Science and Art History. Some fields of CS that I am interested in include Human Computer Interaction, AI, and Software Development. Outside of that, I am on the varsity squash team, love NYC, and am ambidextrous!
+      </p>
+      <b>Resume:</b>
+      <a href="/docs/pranjali_resume.pdf" target="_blank">pdf</a>
+      <div>
+      <img src="/images/clmb sky.jpg" alt="Columbia" width="500" height="333">
+      </div>
+      </div>
+
+      <div id= "projects">
+          <h3>Projects</h3>
+          <p>Here are some projects I have worked on-</p>
+          
+          <div id= "clmbfun">
+              <a href="https://github.com/pranjali-sharma"><b> ColumbiaFun: </b>
+</a> <p>A UI project which lets the user view, search, create, 
+rate, edit fun Columbia-specific activities. Using Flask on 
+the backend to saved data on the server and 
+JS,Bootstrap, HTML, and CSS on the frontend, this is a highly 
+interactive app which includes additions and edits by the user. 
+It implements a navbar functionality for increased ease 
+of access and navigation for the user.</p>
+          </div>
+
+          <div id= "cseg">
+           <a href="https://github.com/cseg-tech/nyc-housingautomation"><b>CSEG Housing Automation: </b></a>
+           <p>This is a group project with Columbia Social Entrepreneurship Group (CSEG)
+            which is a notification system that alerts NYC tenants, 
+            tenant organizers, and other users who sign up for the 
+            service whenever a housing complaint is filed in their building.
+           </p>
+ 
+          </div id= "CLA">
```

Comment:
nit: use consistent indentation in the file. 
so indent <a> tag to match formatting for the rest of the file